2020/join/gatsby-new-join/src/components/business.js

```diff
@@ -0,0 +1,154 @@
+    <a href="https://www.thedp.com/article/2017/11/sponsored-gia-pronto-quiz" target="_blank" rel="noopener noreferrer">Which Gia Pronto Kitchen salad are you? &#8594;</a>
+    <a href="https://www.thedp.com/article/2017/11/sponsored-dim-sum-house-article" target="_blank" rel="noopener noreferrer">Jane G’s Dim Sum House: fresh made-to-order dim sum &#8594;</a>
+
+  </>
+)
+
+const circulation = () => (
+  <>
+    <p>
+      Circulation staffers are the DP's street team, and they are the ones you see handing out papers on Locust Walk and across campus every day.
+          </p>
+    <p>
+      If you're friendly, dependable, and loud &mdash; and want to earn $10 an hour &mdash; this is the department for you.
+      Join an incredible team with a great culture filled with good times and social activities.
+          </p>
+  </>
+)
+
+const Business = () => {
+  const [tab, setTab] = useState(marketing)
+
+  const marketingBG = "linear-gradient(rgba(238, 238, 238, 0.9), rgba(238, 238, 238, 0.9)), url('https://snworksceo.imgix.net/dpn/5803b1e8-72a6-4810-9770-60123f4a066d.sized-1000x1000.jpg')";
+  const consultingBG = "linear-gradient(rgba(238, 238, 238, 0.8), rgba(238, 238, 238, 0.8)), url('https://snworksceo.imgix.net/dpn/7f265244-68c1-4483-b138-8fc49c343444.sized-1000x1000.jpg')";
+  const analyticsBG = "linear-gradient(rgba(238, 238, 238, 0.8), rgba(238, 238, 238, 0.8)), url('https://snworksceo.imgix.net/dpn/68726eb4-7df0-4e26-9d83-3ad542d33846.sized-1000x1000.jpg')";
+  const productLabBG = "linear-gradient(rgba(238, 238, 238, 0.8), rgba(238, 238, 238, 0.8)), url('https://snworksceo.imgix.net/dpn/af307706-8ed7-4389-bb37-283cdb2af2d0.sized-1000x1000.jpg')";
+  const circulationBG = "linear-gradient(rgba(238, 238, 238, 0.8), rgba(238, 238, 238, 0.8)), url('https://snworksceo.imgix.net/dpn/06837fc5-f166-414c-8785-9234e35205b0.sized-1000x1000.jpg')";
+
+  const [background, setBackground] = useState(marketingBG)
+  const [active, setActive] = useState('marketing')
+
+  const changeTab = (tabName) => {
+    switch (tabName) {
+      case 'marketing':
+        setTab(marketing)
+        setActive('marketing')
+        setBackground(marketingBG)
+        break;
+      case 'consulting':
+        setTab(consulting)
+        setActive('consulting')
+        setBackground(consultingBG)
+        break;
+      case 'analytics':
+        setTab(analytics)
+        setActive('analytics')
+        setBackground(analyticsBG)
+        break;
+      case 'product-lab':
+        setTab(productLab)
+        setActive('product-lab')
+        setBackground(productLabBG)
+        break;
+      case 'circulation':
+        setTab(circulation)
+        setActive('circulation')
+        setBackground(circulationBG)
+        break;
+      default:
+    }
+  }
+
+  return (
+    <BgStyle bg = {background}>
```

Comment:
there shouldn't be space `bg = {background}`. Just say `bg={background}`. Also, there should be a `/` to indicate the end of component. So `<BgStyle bg={background} />`